config/crd/default/patches/webhook_in_dynakubes.yaml

```diff
@@ -13,4 +13,4 @@ spec:
           name: dynatrace-webhook
           path: /convert
       conversionReviewVersions:
-      - v1
+      - v1beta1
```

Comment:
does not belong in this PR

Reply:
its already in master, so removing it here does nothing